modules/imgcodecs/src/grfmt_avif.cpp

```diff
@@ -297,16 +299,24 @@ bool AvifEncoder::isFormatSupported(int depth) const {
 
 bool AvifEncoder::write(const Mat &img, const std::vector<int> &params) {
   std::vector<Mat> img_vec(1, img);
-  return writeToOutput(img_vec, params);
+  return writemulti(img_vec, params);
 }
 
 bool AvifEncoder::writemulti(const std::vector<Mat> &img_vec,
                              const std::vector<int> &params) {
-  return writeToOutput(img_vec, params);
+    Animation animation;
+    animation.frames = img_vec;
+    int timestamp = 0;
+    for (size_t i = 0; i < animation.frames.size(); i++)
+    {
+        animation.timestamps.push_back(timestamp);
+        timestamp += 10;
```

Comment:
Looks like timestamps are used differently in writemulti and writeanimation. If I got correctly, timestamps contains frame duration rather then timestamp. In this case, `timestamp` should not be incremented, but set equal to 10 once outside the loop. Also I recommends something like 33ms (fps).

Reply:
@asmorkalov @vrabaud let me change `timestamps` name as `durations` means how much ms the frame is shown

Reply:
i removed the line `timestamp += 10;`